app-backend/tool/src/main/scala/eval/TileProvider.scala

```diff
@@ -0,0 +1,53 @@
+package com.azavea.rf.tool.eval
+
+import geotrellis.raster._
+
+import java.security.InvalidParameterException
+
+
+case class Buffers(
+  tl: Tile,
+  tm: Tile,
+  tr: Tile,
+  ml: Tile,
+  mr: Tile,
+  bl: Tile,
+  bm: Tile,
+  br: Tile
+)
+
+case class TileProvider(centerTile: Tile, buffers: Option[Buffers], options: TileProvider.Options = TileProvider.Options.DEFAULT) {
```

Comment:
`TileProvider` implies a pattern thats not really in use here. This looks more like `TileWithNeighbors`, a result of some pre-operation join.